src/NuGet.Server/NuGet.Server.nuspec

```diff
@@ -18,6 +18,7 @@
     <dependencies>
       <dependency id="NuGet.Core" version="[2.11.1,)" />
       <dependency id="Microsoft.Web.Xdt" version="[2.1.1,)" />
+	  <dependency id="NuGet.Server.Core" version="[2.11.1,)" />
```

Comment:
alignment, and I think these packages should have a 1:1 dependency instead of an open ended. They only go together as a wave.